experimental/fRIP/fRIP_MPRNA_LibraryWorkupProtocol.md

```diff
@@ -0,0 +1,115 @@
+4. Incubate at 65°C for 5 minutes, then snap-cool on ice  
+5. Make RT master mix:  
+
+- 4 uL 5x SuperScript IV Buffer  
+- 1 uL 100 mM DTT  
+- 1 uL RNaseOUT  
+- 1 uL SuperScript IV  
+
+6. Add 7 uL of RT master mix to each PCR tube  
+7. Run the following program in the thermal cycler:  
+
+- 23°C for 10 minutes  
+- 55°C for 10 minutes  
+- 80°C for 10 minutes  
+
+# Amplicon library PCR  
+Because of how many reads machines like MiSeq and NextSeq achieve per run, you'll likely want to pool multiple amplicon libraries together in one run when it comes time to sequence, meaning you'll want to use primers that will put different indices on each amplicon library in a given group that you're pooling. As of the writing of this protocol as a markdown file (2020-05-01), all of our lab's oligo pools are designed such that the following primer sequences will fit the purpose of this PCR:  
+
+**Fwd primer:** caagcagaagacggcatacgagatXXXXXXgtgactggagttcagacgtgtgctcttccgatctACTGGCCGCTTCACTG (where XXXXXX is the index; JR4-17 and JR45 in our primer database are all versions of this primer with different indices)  
+**Rev primer:** AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT (named JR3 in our primer database)  
+
+Once you've chosen primers for each library, make a mix of each primer pair where each primer is at a concentration of 2 uM. Then, for each library combine the following in a PCR tube:  
+
+- 10 uL of cDNA (save other 10 uL)  
+- 5 uL 2 uM primer mix  
+- 35 uL of master mix (1x recipe):  
+  26.5 uL nuclease-free water  
+  5 uL 10x Pfu Ultra Buffer  
+  2.5 uL 10 mM dNTP mix  
+  1 uL Pfu Ultra II (Agilent 600672)  
+  *NOTE: In some situations, I have found that adding 10 uL of NEB 5x High GC Enhancer (included with Q5 polymerase (NEB M0491)) to these reactions helps a little with representation. If you want to use it, the 1x master mix recipe will only have 16.5 uL of water.*  
+
+Run the following program in the thermal cycler:  
+
+- 95°C for 2:00  
+---  
+- 95°C for 0:30 |  
+- 55°C for 0:30 | **24-30 cycles**  
+- 72°C for 0:30 |  
+---  
+- 72°C for 10:00  
+- Hold at 4°C  
+
+Run 1 uL of the PCR on the Bioanalyzer (HS DNA Kit, Agilent 5067-4626) to check for a peak of the correct size. If you don't see a strong enough peak at the correct size, run the PCR for several more cycles and check again.  
+
+- *NOTE: Almost always, I find that 24 cycles is sufficient to get a strong peak, so I would start there and only add more cycles if needed.*  
+
+# Amplicon library clean-up (0.6X/1.6X/1.0X triple SPRI cleanup)  
+Before starting, warm AMPure XP beads (Beckman Coulter A63881) to room temp for 30 minutes, and make fresh 70% ethanol. If you have more than 8 samples, transfer them to a 96-well plate that fits on the magnet you'll be using (with 8 or fewer samples, you can just use PCR strips).  
+
+**SPRI 0.6X**  
+
+1. Add 0.6X volume AMPure beads (30 uL) and mix by pipetting (gently to minimize foaming).  
```

Comment:
little more detail and product id

Reply:
The product info for the AMPure beads is included a few lines above this, right below the "Amplicon library cleanup" heading.